selftests/check.py

```diff
@@ -539,9 +553,12 @@ def main():
     # Job execution
     # ========================================================================
     config = {'core.show': ['app'],
-              'run.test_runner': 'nrunner'}
+              'run.test_runner': 'nrunner',
+              'run.journal.enabled': True}
     with Job(config, suites) as j:
-        return j.run()
+        exit_code = j.run()
+        print_failed_tests(j.logdir)
```

Comment:
Nipick: this and the following line can be put outside of the context manager (and probably should as an extra guarantee the job is finished).